lib/blueprinter/view.rb

```diff
@@ -2,18 +2,22 @@ module Blueprinter
   # @api private
   DefinitionPlaceholder = Struct.new :name, :view?
   class View
-    attr_reader :excluded_field_names, :fields, :included_view_names, :name, :transformers, :definition_order
+    attr_reader :excluded_field_names, :fields, :included_view_names, :name, :view_transformers, :definition_order
 
     def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [], transformers: [])
       @name = name
       @fields = fields
       @included_view_names = included_view_names
       @excluded_field_names = excluded_view_names
-      @transformers = transformers.empty? ? Blueprinter.configuration.default_transformers.clone : transformers
+      @view_transformers = transformers
       @definition_order = []
       @sort_by_definition = Blueprinter.configuration.sort_fields_by.eql?(:definition)
     end
 
+    def transformers
+      view_transformers.empty? ? Blueprinter.configuration.default_transformers.clone : view_transformers
```

Comment:
Now that we've separated these out, I don't think we need the `.clone` right?

Reply:
Indeed! Good catch, fixed in 062456a